### ISSUE TITLE
Multiple Wikipedia links

### ISSUE THREAD

**Issue by zstadler** [IsraelHikingMap/Site#1165]
## What I expect to happen
The details panel should show Wikipedia links for the selected language

## What really happened
Both the English and Hebrew Wikipedia icons are shown for [יד אבשלום](https://israelhiking.osm.org.il/poi/OSM/way_32591134?language=he)

![image](https://user-images.githubusercontent.com/1304610/74910124-76cb0980-53c2-11ea-8040-6413c234b332.png)

**Comment by HarelM** [comment]
When refactoring/redesigning the merge I have started to add a website language identifier.
After a lot of changes back and forth I have decided that this is the right way to handle this and links to other sites are not language specific.
There is a comment about it here in the discussion:
https://wiki.openstreetmap.org/wiki/Talk:Key:website
But in general I would like to avoid adding the logic of language to website links.
Bottom line, I believe that fixing this does not worth the extra complexity in the merge algorithm (which is already very complex) and there are places where the website language is unknown (for example when added by a user).
I'm in favor of closing this.

**Comment by valleyofdawn** [comment]
I think this can be reconsidered.
It's a frustrating user experience to have two identical icons, click on one hoping to go to the Hebrew site, reach the English site and then go back to tap the other.
This needs not be applied to all websites, only for Wikipedia ones - which are the great majority.
It's easy to identify if a user-entered wiki site is in English or Hebrew  -- one URL starts with "en", the other with "he".
Alternatively, the wiki icon should identify the language with an "E" or an "ע" on top.

**Comment by HarelM** [comment]
Looking at the statistics of the site, almost no one is using these links. There's no good reason to invest time in this.

**Comment by HarelM** [comment]
This issue was recently reported again by @amiad.
Since all the conversion logic has now moved to the client side it is possible to decide which links to present since all the information is there.
Having said that, this will not be a general solution for language specific links but rather a solution only for the wikipedia tags.
Since the usage of the site in English is low and isn't planned to be larger I have no issues with patching only this part.
Relevant conversion code can be found here, and adding a check for `<lang>.wikipedia` shouldn't be a big change.
https://github.com/IsraelHikingMap/Site/blob/f7382bd1e72ce77325dd31aa080d5b1c90f52569/IsraelHiking.Web/src/application/components/sidebar/publicpoi/public-poi-sidebar.component.ts#L192
